ui/src/pages/Admin/Edit/UploadStory.tsx

```diff
@@ -618,7 +650,7 @@ export const UploadStory: React.FC<StoryProps> = ({
                 <PrimaryButton
                   text={submitButtonText}
                   onClickFunction={(e) => handleSubmit(e)}
-                  isDisabled={state.disabled || !hasAllRequiredFields}
+                  isDisabled={state.disabled || !hasAllRequiredFields || !hasFormChanged}
```

Comment:
bro I need a truth table to decipher this

Reply:
throwback to https://en.wikipedia.org/wiki/Propositional_calculus, the GOAT calculus